src/tests/ftest/cart/protocol_info.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * (C) Copyright 2018-2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+
+#include <cart/api.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#define NWIDTH 20
+
+static int
+print_info(const char *info_string)
+{
+	struct crt_protocol_info *protocol_infos = NULL, *protocol_info;
+	int                       ret;
+
+	ret = crt_protocol_info_get(info_string, &protocol_infos);
+	if (ret != DER_SUCCESS) {
+		fprintf(stderr, "crt_protocol_info_get() failed (%d)\n", ret);
+		return ret;
+	}
+	if (protocol_infos == NULL) {
+		fprintf(stderr, "No protocol found for \"%s\"\n", info_string);
+		return -DER_NOTSUPPORTED;
+	}
+
+	printf("--------------------------------------------------\n");
+	printf("%-*s%*s%*s\n", 10, "Class", NWIDTH, "Protocol", NWIDTH, "Device");
+	printf("--------------------------------------------------\n");
+	for (protocol_info = protocol_infos; protocol_info != NULL;
+	     protocol_info = protocol_info->next)
+		printf("%-*s%*s%*s\n", 10, protocol_info->class_name, NWIDTH,
+		       protocol_info->protocol_name, NWIDTH, protocol_info->device_name);
+
+	crt_protocol_info_free(protocol_infos);
+
+	return DER_SUCCESS;
+}
+
+int
+main(int argc, char *argv[])
+{
+	const char *info_string = NULL;
+	int         crt_ret;
+
```

Comment:
either crt_init() or d_log_init() here (with a corresponding fini at the end) to get the logging capabilities. almost all of our tests use gurt level macros to print (D_ERROR/D_WARN/D_INFO/D_DEBUG). Makes it easier to find logs in the proper places in CI later.

We also have DBG_PRINT() macro as part of crt_utils which duplicates to cart logs and stderr/stdout, just need to call this function then to initialize it:

 void crtu_test_init(d_rank_t rank, int num_attach_retries, bool is_server, bool assert_on_error);